### PR TITLE
feat: replace hibernate modal with confirmation modal

### PR DESCRIPTION
**Pull request by arunjaindev**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -337,6 +337,7 @@ export default function App() {
                                     <div id="full-screen-modal" />
                                     <div id="visible-modal" />
                                     <div id="visible-modal-2" />
+                                    <div id="backdrop" />
```
Comment (AbhishekA1509) [inline src/App.tsx:340]: Can we rename since its too generic

File: src/components/common/navigation/NavigationRoutes.tsx [hunk 2]
```diff
@@ -380,7 +380,7 @@ export default function NavigationRoutes() {
                 isSuperAdmin,
             }}
         >
-            <main className={`${_isOnboardingPage ? 'no-nav' : ''}`}>
+            <main className={`${_isOnboardingPage ? 'no-nav' : ''}`} id='devtron-base-main-identifier'>
```
Comment (AbhishekA1509) [inline src/components/common/navigation/NavigationRoutes.tsx:383]: Please use constant from fe-common

File: src/components/common/navigation/NavigationRoutes.tsx [hunk 3]
```diff
@@ -380,7 +380,7 @@ export default function NavigationRoutes() {
                 isSuperAdmin,
             }}
         >
```
Comment (AbhishekA1509) [inline src/components/common/navigation/NavigationRoutes.tsx:382]: We can directly use _isOnboardingPage ? 'no-nav' : ''

File: src/components/app/details/appDetails/AppDetails.tsx [hunk 4]
```diff
@@ -58,14 +60,14 @@ import { getAppConfigStatus, getAppOtherEnvironmentMin, stopStartApp } from '../
 // @ts-check
 import AppNotDeployedIcon from '../../../../assets/img/app-not-deployed.png'
```
Comment (AbhishekA1509) [inline src/components/app/details/appDetails/AppDetails.tsx:61]: Suggestion: Can we use alias import?

**Review by AbhishekA1509 — CHANGES_REQUESTED** [review]

**Review by AbhishekA1509 — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Review by AbhishekA1509 — APPROVED** [review]

**Pull request merged**